packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -92,8 +92,7 @@ function CardContainerInternal(props: CardContainerProps) {
   } = props
   const { NoContentPlaceholder } = useQueryVisualizationContext()
   const queryContext = useQueryContext()
-  const { queryMetadataQueryOptions } = queryContext
-  const { data: queryMetadata } = useSuspenseQuery(queryMetadataQueryOptions)
+  const { data: queryMetadata } = useSuspenseGetQueryMetadata()
```

Comment:
in tests, we could then just mock `useSuspenseGetQueryMetadata` instead of either trying to mock react-query or creating a mock queryFn to pass to react-query.